docker/openqa_data/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM opensuse:13.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com> Josef Skladanka <jskladan@redhat.com>
+
+RUN zypper --non-interactive --gpg-auto-import-keys in ca-certificates git wget
```

Comment:
It's not necessary to use --gpg-auto-import-keys here, but it's necessary to use it when installing OpenQA. I don't know how to handle certificates of OpenQA repo (where to get that certificate and how to import it).